cmds/tui.c

```diff
@@ -167,15 +168,36 @@ static const char *graph_field_names[NUM_GRAPH_FIELD] = {
 	"TOTAL TIME", "SELF TIME", "ADDRESS",
 };
 
-static const char *graph_field_help[] = {
+#define NUM_REPORT_FIELD 9
+
+static const char *report_field_names[NUM_REPORT_FIELD] = {
+	"TOTAL TIME", "TOTAL AVG", "TOTAL MIN", "TOTAL MAX",
+	"SELF TIME", "SELF AVG", "SELF MIN", "SELF MAX",
+	"CALL",
+};
+
+static const char *field_help[] = {
 	"DOWN/UP ARROW Move down/up",
 	"j/k           Move down/up",
 	"Enter         Apply checked fields",
 	"SPACE         Check or uncheck a field",
 	"f/q           Close the window without any changes",
 };
 
-static const char *report_sort_key[] = { OPT_SORT_KEYS, "self", "call" };
+enum mode {
```

Comment:
This can be ignored but I just think `enum tui_mode` is better here. `enum mode` is too generic.

Reply:
@honggyukim Thanks for your comment.
I applied it.